src/app/zap-templates/templates/app/MatterIDL.zapt

```diff
@@ -63,14 +65,14 @@
       {{/unless}}
     {{/enabled_attributes_for_cluster_and_side~}}
 
+  {{~/if}}
   {{~!--Global attributes: spec 7.13 describes these as mandatory --}}
   readonly attribute command_id generatedCommandList[] = 65528;
   readonly attribute command_id acceptedCommandList[] = 65529;
-  readonly attribute event_id eventList[] = 65530;
+  readonly attribute optional event_id eventList[] = 65530;
```

Comment:
eventList is not optional, right?

Reply:
eventList is not implemented in v1.0 or v1.1. It cannot be mandatory.

Reply:
It's mandatory in the spec and in the XML.

There's a difference between "mandatory feature not yet implemented" and "optional feature".

Reply:
I am approaching it from a perspective of "what would an automated tool validating things" do (it so happens that this is how I found this): if we validate everything that says mandatory as existing ,this basically fail.

I got this for eventList and for UnitTestingCluster where we have 2 fields that are mandatory but intentionally error out on reads (boolean and global error fields). I special-cased the unit test since that seemed reasonable to do, however for event list optional seems the correct thing to do as it is the current behavior.

Reply:
I guess my statement is: for the purposes of `.matter` there is no difference between "mandatory but not implemented" and "optional" ... I am not sure "mandatory but not implemented" even makes sense practically.

Reply:
Realistically, leaving it out completely is better than claiming it's "optional" when it's not....

Reply:
Ok, I'll have to think about this. Maybe add another quality of "provisional" (not-implemented sounds odd).

Ideally .matter should reflect what zap sees in the world, but if I do that here it will mark all global attributes 'optional' and that seems worse, and if I fix that I am afraid it will have ripple effects.

Reply:
Created issue and updated to not make eventList optional.